webextensions/chrome/background.js

```diff
@@ -368,27 +369,28 @@ const ThinBridgeTalkClient = {
     this.save();
   },
 
+  checkRedirectIntervalLimit(tabId, url) {
+    const intervalLimit = 1000;
+    let skip = false;
+    for (const key in this.recentRedirections) {
+      if (Math.abs(Date.now() - this.recentRedirections[key].time) > intervalLimit)
+        delete this.recentRedirections[key];
+    }
+    const recent = this.recentRedirections[tabId];
+    if (recent && recent.url === url) {
+      console.log(`Redirection for same URL and same tabId already occurred in ${intervalLimit} msec. Skip it.`);
+      skip = true;
+    }
+    this.recentRedirections[tabId] = { tabId: tabId, url: url, time: Date.now() }
```

Comment:
`tabId` in value seems to be needless because it duplicates with table key (`tabId`).

```suggestion
    this.recentRedirections[tabId] = { url: url, time: Date.now() }
```

Reply:
I prefer containing `tabId` here for passing & referring this object alone at somewhere.

Reply:
I see, I agree with leaving it as is.